### ISSUE TITLE
Project/Kanban setup

### ISSUE THREAD

**Issue by denscholar** [denscholar/space-traveller-app#25]
### Project link: https://github.com/denscholar/space-traveller-app/projects/1

Please, note that we are two on this team for this project.; therefore, we have archived all tasks meant for the third person.  

Thank you.

**Comment by Rustamxon7** [comment]
Hi Team, 

Good job so far!

![emojibest_com_AnimatedSticker](https://user-images.githubusercontent.com/69011963/157894616-c6ae63a9-a266-4fbf-a9bd-2a95a94dfc0c.gif)


### HIGHLIGHTS!

- No linters errors.💯
- Descriptive Readme.md. 💯
- Awesome UI with additional features

There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

- [x] **GOOD JOB 😃👍🎉👏** Please kindly fix the audio in your video presentation and record it again 🎥. I think your microphone is not working 🎤.  I'd suggest also add a highlight about a piece of the code the you find interesting or is proud of making in this project 🍀

_Also check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rustamxon7** [comment]
1. - [ ] When it comes to tasks there are only 16 tasks but there must be 
 17 tasks please add one more task for it. 


2. - [ ] All tasks should be divided for each team member!
I can see on your Project some places didn't assign.

|---------------------------------------------------------|
<img width="359" alt="Screen Shot 2022-03-15 at 1 37 16 AM" src="https://user-images.githubusercontent.com/69011963/158258183-348ba9c9-0dc3-4042-b252-b02f40588655.png">

**Comment by rukundoeric** [comment]
Hi @denscholar,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------

**Comment by denscholar** [comment]
@Rustamxon7 Thank you for the feedback. However, it would be nice you be more careful and thorough when providing feedback knowing fully well how much time was invested to get things like these done. I was surprised when you mentioned this:
![Capture](https://user-images.githubusercontent.com/48631109/158262892-e6b903e3-e794-4d69-be03-d3a5298b5fb0.PNG)

Thank you.

**Comment by Rustamxon7** [comment]
> @Rustamxon7 Thank you for the feedback. However, it would be nice you be more careful and thorough when providing feedback knowing fully well how much time was invested to get things like these done. I was surprised when you mentioned this: ![Capture](https://user-images.githubusercontent.com/48631109/158262892-e6b903e3-e794-4d69-be03-d3a5298b5fb0.PNG)
> 
> Thank you.

Yes sorry about that issue! You are right there is not that kind of requirement in this project. It was my issue that I forgot to remove this line while I'm copying from old code! Hope you understand. 😊
